testing/trino-faulttolerant-tests/pom.xml

```diff
@@ -185,6 +185,12 @@
             <groupId>io.trino</groupId>
             <artifactId>trino-hdfs</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.qubole.rubix</groupId>
```

Comment:
why? revert

Reply:
This is a bit of an awkward dependency situation. `com.qubole.rubix:rubix-presto-shaded` and `trino-filesystem-cache-alluxio` both share some classes (definitely `com.github.ishugaliy:allgood-consistent-hash`, and maybe some `dropwizard` dependency), and we get duplicate classes on the class path if both jars are included. Since the Rubix dependency is shaded I don't know what's the best approach:
* We could keep it as is.
* We could skip the Alluxio dependency instead.
* We could exclude the conflicting dependencies from Alluxio, and rely on the classes being provided by the Rubix dependency.

There should be little harm in excluding the Rubix dependency from the fault tolerant tests, as it does not look like we use it there.

Reply:
@jkylling we plan to drop rubix all along as it's not working in production cases and it's basically abandondware. Please create a preparatory commit (1st) that drops rubix altogether